src/app/components/header/header.component.html

```diff
@@ -1,17 +1,24 @@
 <header class="header">
-  <div class="header-content">
-    <div class="navigation">
-      <a routerLinkActive="active-link" routerLink="/products">Products</a>
+    <div class="header-content">
+        <div class="navigation">
+            <a *appRole="'user,subscribed'" routerLinkActive="active-link" routerLink="/products">Products</a
```

Comment:
it is better to use the service to obtain information about the user's role

Reply:
Sorry, I see you are using subscribe in a directive, but in this case you don't need input.